style.css

```diff
@@ -87,3 +87,229 @@ body {
   font-family: 'Inter', sans-serif;
   font-weight: 400;
 }
+
+.arrowDown {
+  margin-left: auto;
+  margin-right: auto;
+  display: block;
+  width: 10%;
+}
+
+.myRecentWork {
```

Comment:
As specified in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20in%20%22Works%22%20section%20you%20should%20use%20CSS%20Grid%20and%20Flexbox%20combined%20(it%20is%20mandatory%20that%20you%20use%20both).%20Use%20CSS%20Grid%20in%20the%20main%20container%20that%20contains%20all%20of%20the%20cards%20and%20use%20Flexbox%20to%20lay%20out%20an%20individual%20card.), for this section alignment you are required to use `CSS Grid` , please make sure your section is aligned with Grid and your cards with Flex :frog: 

[Here you can find some tricks and tips about CSS Grid](https://css-tricks.com/snippets/css/complete-guide-grid/) that can help you with the implementation this requirement :+1: